src/bin/migrate/migrations/alpha14-to-alpha15.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  type JsxAttribute,
+  SyntaxKind,
+  type Project,
+  type SourceFile,
+} from 'ts-morph';
+import { isDesignSystemImport } from '../helpers';
+
+export default function migration(project: Project) {
+  const files = project.getSourceFiles();
+  const sourceFiles = files.filter((file) => !file.isDeclarationFile());
+
+  sourceFiles.forEach((sourceFile) => {
+    updateStatusProp({ file: sourceFile }); // https://github.com/chanzuckerberg/edu-design-system/pull/1973
```

Comment:
@jeremiah-clothier FYI For this one-off, custom migration, i've put this into one migration file for now. We shouldn't need this particular one as a common case. 

The pattern in this case would be that any X-to-Y.ts file would re-use the common cases, and include the one-off transforms there as well to avoid confusion about longevity.